pkg/util/kubernetes/apiserver/hpa_controller.go

```diff
@@ -159,13 +164,16 @@ func (h *AutoscalersController) syncHPA(key interface{}) error {
 		newMetrics := h.hpaProc.ProcessEMList(emList)
 		// The syncWPA can also interact with the overflowing store.
 		if len(newMetrics)+h.metricsProcessedCount > maxMetricsCount {
-			log.Warnf("Currently processing %d metrics, skipping %s/%s as we can't process more than %d metrics",
```

Comment:
I think we should still keep the log, for users that don't rely on events, they would lose the information.